src/interface.rs

```diff
@@ -1,5 +1,5 @@
 use defmt::trace;
-use embedded_hal_async::spi::SpiBus;
+use embedded_hal_async::spi::SpiDevice;
 
 use crate::mod_params::RadioError;
```

Comment:
Could you switch to ` use crate::mod_params::RadioError::{self, SPI}`;

Reply:
Done